apollo-core/src/test/java/com/spotify/apollo/core/ServiceImplTest.java

```diff
@@ -366,16 +375,8 @@ public void testSignalShutdown() throws Exception {
     Service service = ServiceImpl.builder("test").build();
     try (final Service.Instance instance = service.start("--syslog=false")) {
 
-      // Force predictable concurrency of signal and main loop start
-      final CountDownLatch countDownLatch = new CountDownLatch(1);
-      instance.getExecutorService().submit(() -> {
-        countDownLatch.await();
-        Thread.sleep(200);
-        instance.getSignaller().signalShutdown();
-        return null;
-      });
+      instance.getSignaller().signalShutdown();
```

Comment:
Yes. I didn't see much added value in doing shutdown and wait on separate threads, though, so I simplified the test case. Do you think there is value there?